backend/src/main/scala/bloop/logging/ProcessLogger.scala

```diff
@@ -41,15 +41,24 @@ object ProcessLogger {
     val outputStream = new OutputStream {
       private[this] var dirty = false
       private[this] val buffer = new ByteArrayOutputStream
+      private[this] var sawR = false
 
       override def write(x: Int): Unit = synchronized {
         dirty = true
-        if (x == '\n') flush()
-        else buffer.write(x)
+        if (x == '\r') sawR = true
```

Comment:
How will this affect Windows users when they run and test in Windows machines? I wonder if it wouldn't be better to enable this only for our CI? I'm afraid about the implications of this change regarding correctness and performance (running a verbose application with tons of logs).

Reply:
I'll experiment more today and try to remove it, but it seemed to me this was required for having the `ProcessLogger` work on Windows...

Reply:
The question is, what message should the logger receive when someone does:
`System.out.println("foo")` ?
Should it contain `foo` (as it is currently in this PR), or
`foo\r` (as it is in `master`)

Reply:
Fixed as discussed today. We just don't care about `\r` in the tests.